tools/dependencies/known-dependencies.txt

```diff
@@ -253,3 +253,6 @@ xmlbeans-3.1.0.jar
 xmlenc-0.52.jar
 xz-1.0.jar
 zookeeper-3.4.14.jar
+moco-core-1.2.0.jar
+moco-runner-1.2.0.jar
```

Comment:
Hi @baisui1981 , you need to check why these 2 testing frameworks are included in the distributed binary tar

Reply:
tks，i  forgot to remove it.